bin/oneoff/migrate_workshop_survey_data.rb

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env ruby
```

Comment:
Does this belong in `bin/oneoff/data_fix` or `bin/oneoff/backfill_data` (presumably not `bin/oneoff/wipe_data`)? Trying to keep that growing directory more organized...